devdoc/docs/ContributeDocs.md

```diff
@@ -0,0 +1,394 @@
+        user questions about the feature to pre-empt confusion.
+
+4.  **Additional Resources**
+
+    -   **Links to related documentation:** Include links to relevant
+        documentation for features that interact with this new addition.
+
+    -   **Tutorials:** Consider creating tutorials for a more
+        interactive learning experience.
+
+    -   **Videos:** Consider requesting that a video be made to provide
+        a more interactive visual learning experience. You should
+        provide a simple script or outline of what should be shown in
+        the video.
+
+### General Tips
+
+-   **Target your audience:** Tailor the level of detail and technical
+    jargon you use based on whether the documentation is for developers
+    or end-users.
+
+-   **Clarity and Conciseness:** Use clear, concise language and
+    maintain a consistent structure for easy navigation. Always use
+    present tense, active voice. Remember, you’re writing for the
+    open-source community, not academics, so keep it simple and
+    straight-forward. See the HPCC Style Guide for additional guidance.
+
+-   **Visual Aids:** Screenshots, diagrams, and flowcharts can
+    significantly enhance understanding. A picture can communicate
+    instantly what a thousand words cannot.
+
+-   **Maintain and Update:** Regularly review and update documentation
+    as the feature evolves or based on user feedback.
+
+By following these guidelines and including the required and optional
+components, you can create comprehensive documentation that empowers
+users and streamlines the adoption of your new software feature.
+
+### Placement
+
+In general, it makes sense to keep your documentation near the code. For
+example, a document about ECL Agent should go in the ECLAgent folder.
+
+However, there are times where that is either not possible or a document
+may cover more than one component. In those cases, there are a few
+options as shown below.
+
+#### Other Folders
+
+**DevDocs**: This is a general folder for any developer document.
+
+**Docs**: This is a folder for documents about documentation.
+
+**Userdoc**: This is a collection of docs targeted toward the end-user
+rather than developers.
+
+**Userdoc**/**troubleshoot**: Information related to troubleshooting
+particular components
+
+**userdoc**/**azure**: Useful information about Azure Cloud portal and
+cli
+
+**userdoc**/**roxie**: Useful information for running roxie
+
+**userdoc**/**thor**: Useful information for running thor
+
+**userdoc**/**blogs**: COMING SOON: Location and storage of original
+text for blogs. It also has docs with guidelines and instructions on
+writing Blogs
+
+#### Pull Requests
+
+You can include your documentation with your code in a Pull Request or
+create a separate Jira and Pull Request for the documentation. This
+depends on the size of the code and doc. For a large project, a separate
+Pull request for the documentation is better.
+
+# Writing Style Guide
```

Comment:
This should probably be a separate document.  It is describing the craft of writing documentation, rather than what documentation should be written, and where you find the different kinds of documentation.

Reply:
I had originally had it as a separate doc. I will do that again